crates/chia-protocol/src/program.rs

```diff
@@ -68,6 +68,12 @@ impl Program {
         self.0.into_inner()
     }
 
+    pub fn get_tree_hash(&self) -> crate::Bytes32 {
```

Comment:
why did you move this?

Reply:
I made it public so I could use it in the new api function